.github/workflows/test.yml

```diff
@@ -360,3 +360,21 @@ jobs:
       working-directory: ./vscode
     - run: vsce package
       working-directory: ./vscode
+
+  std-lib-tests:
+    name: Stdlib
+    runs-on: solang-ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
```

Comment:
I don't think it needs submodules. Removing these two lines will make it faster as solc does not have to be cloned.